src/utils/module.h

```diff
@@ -75,7 +75,7 @@ class CModule
 		for (size_t i = 0; i < m_size; i++)
 		{
 			size_t Matches = 0;
-			while (*(pMemory + i + Matches) == pData[Matches] || pData[Matches] == '\x2A')
+			while (Matches < iSigLength && *(pMemory + i + Matches) == pData[Matches] || pData[Matches] == '\x2A')
```

Comment:
Is this meant to be `(less than sig length AND memory matches) OR sig is wildcard`… or is it `less than sig length AND (memory matches OR sig is wildcard)`? Because currently it’s the first one.

Regardless should probably make it explicit to avoid confusion.